src/sentry/static/sentry/app/utils/auth.jsx

```diff
@@ -0,0 +1,16 @@
+import cookies from './cookies';
+
+export default {
+  signOut: function () {
```

Comment:
Seems fine – I would just comment here _why_ this is an iframe POST and not an API request (for someone who might come along later and might not easily discover all the comments/activity in this PR).